README.md

```diff
@@ -1,8 +1,5 @@
 # Bethink design system
 
-## Library info
-For quicker development during initial phase of Design System we decided to publish this repo as is - not as a [Vue library](https://cli.vuejs.org/guide/build-targets.html#library).
-
 ### Typescript
```

Comment:
We serve parsed code now, do we still need this section?

Reply:
We need to export `initialize` and `initializeWithDomWatcher` from library to achieve that

Reply:
we do not, hover I suggest to keep it here until we make some webpack usage document on conflu

Reply:
this makes platform build faster and without hacks. IMO it's worth doing it the right way now.